history/taskana-simplehistory-rest-spring-example/pom.xml

```diff
@@ -1,99 +1,111 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We don't have a formatter yet for our *.xml files. So this is fine for now.

What did you exactly change?